versioned/tests/src/main/java/org/projectnessie/versioned/testworker/BaseContent.java

```diff
@@ -22,7 +22,9 @@ enum Type {
     /** Content type with on-reference state. */
     ON_REF_ONLY,
     /** Content type with on-reference state and mandatory global state. */
-    WITH_GLOBAL_STATE
+    WITH_GLOBAL_STATE,
+    /** Content type with on-reference state and mandatory per-content-id state. */
+    WITH_PER_CONTENT_STATE
```

Comment:
Per content ID _and_ per commit?... A simple per content ID feels like "global" :thinking:

Reply:
Renamed to `WITH_ATTACHMENTS`